[author] Add mathjax-mhchem w/ git auto-update via single package.json

mhchem opened the pull request:
This is moving MathJax/mhchem to cdnjs, because cdn.mathjax.org will be closed (https://www.mathjax.org/cdn-shutting-down/).

This is adding a new library via a single package.json with auto-update config.

I intentionally do _not_ include a .min.js file, because MathJax loads its extension via `extensions: ["[Contrib]/mhchem/mhchem.js"]` and `mhchem.js` always was the minified file. Users are used to request/include `mhchem.js` in order to get a minified file.

The extension is popular. It is used by StackExchange.

# Profile of the lib
 * Git repository (required): https://github.com/mhchem/MathJax-mhchem
 * Official website (optional, not the repository): https://mhchem.github.io/MathJax-mhchem/
 * License and its reference: Apache 2.0
 * Popularity: Used by chemistry.stackexchange.com with 50k visits per day (https://chemistry.meta.stackexchange.com/a/88/24052, http://stackexchange.com/sites#traffic)

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

PeterDaveHello commented on ajax/libs/mathjax-mhchem/package.json:
```diff
@@ -0,0 +1,29 @@
+        "files": [
+          "mhchem.js",
+          "License.txt",
+          "tests/**/*"
```
Tests and license should not be hosted on CDN, please remove it.

pvnr0082t commented on ajax/libs/mathjax-mhchem/package.json:
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mathjax-mhchem",
+  "filename": "mhchem.js",
```
`"filename"` should point to the main minified file. It would be `mhchem.min.js`. Thanks.

pvnr0082t replied:
@mhchem Please modify `filename` field point to the main minified file. Thanks.

mhchem commented:
I added "author". (I had a "contributors" field before, because there is more than one author. But your bot complained. Strange.)

When adding "homepage", I got the error "Homepage doesn't need to be set if it's the same as repository.", so I removed it again.

pkra commented:
:+1: this is an important extension to MathJax that is used on, e.g., stackexchange sites. 

cc @PeterDaveHello

pvnr0082t commented on ajax/libs/mathjax-mhchem/package.json:
```diff
@@ -0,0 +1,28 @@
+      {
+        "basePath": "",
+        "files": [
+          "mhchem.js"
```
It's more flexible to use `"mhchem*"`.

mhchem commented:
Added "homepage" (I would have made a distinction between documentation and homepage. But I don't mind.)
Changed "files" to "mhchem*".

mhchem commented:
@pvnr0082t mhchem.js is the minfied file, so "filename": "mhchem.js" is correct.

I intentionally do not include a .min.js file, because MathJax loads its extension via extensions: ["[Contrib]/mhchem/mhchem.js"] and mhchem.js always was the minified file. Users are used to request/include mhchem.js in order to get a minified file and I don't want to force them to request a different file name just because they request it from cdnjs.

PeterDaveHello commented:
@mhchem nice!

pvnr0082t commented:
@mhchem Okay. Sorry for my mistake.

PeterDaveHello commented:
@pvnr0082t since there is only single file, I'll prefer to ask for its assets added next time, not a problem to add it on gui at all, thanks.

PeterDaveHello commented:
Thanks all!
